README.md

```diff
@@ -15,7 +15,24 @@ In contrast, our philosophy is rooted in simplicity:
 2. Prioritize clarity
 3. Embrace simplicity in design
 
-We believe this approach is the best way to build a client that is both fast and resilient. By adhering to these principles, we will be able to iterate fast and explore next-generation features early, either from the Ethereum roadmap or from innovations from the L2s.
+We believe this approach is the best way to build a client that is both fast and resilient. By adhering to these principles, we will be able to iterate fast and explore next-generation features early, either from the Ethereum roadmap or from innovations from the L2s. 
+
+## Quick Start
+![Demo][./demo.png]
+You'll need docker, docker compose and rust 1.81.
+### Mac
```

Comment:
Maybe just a link to the kurtosis install page (since it has a tab for every os)